pyproject.toml

```diff
@@ -47,9 +47,13 @@ local_scheme = "no-local-version"
 line-length = 88
 target-version = ['py38']
 extend-exclude = '''(
-  (_pb2.pyi?|_pb2_grpc.pyi?)$
+  ^.*(_pb2.pyi?|_pb2_grpc.pyi?)$
 )'''
 
+[tool.isort]
+profile = "black"
+extend_skip_glob = ["*_pb2.py*", "_pb2_grpc.py*"]
+
```

Comment:
The `profile = "black"` is important here, otherwise `isort` might conflict with `black`, resulting in breaking pre-commit.ci autofix.